ogr/services/github/project.py

```diff
@@ -178,6 +178,33 @@ def get_branches(self) -> List[str]:
     def get_description(self) -> str:
         return self.github_repo.description
 
+    def add_to_collaborators(self, user, access_level):
+        """
+        10 => Pull
+        20 => Triage
+        30 => Push
+        40 => Admin
+        50 => Maintain
+        """
+        access_dict = {
+            10: "Pull",
+            20: "Triage",
+            30: "Push",
+            40: "Admin",
+            50: "Maintain",
+        }
```

Comment:
Can we have some enum instead?

e.g.
```python
my_project.add_to_collaborators("the-user", UserAccess.guest)
```

- It's more readable.
- We can have symlinks (e.g. `UserAccess.guest == UserAccess.pull`).